src/components/line-chart/line-chart.component.html

```diff
@@ -1,8 +1,20 @@
-<button (click)="toggleMeasure()">Toggle Measure</button>
-<app-line-chart-visualization role="img"
-                              [attr.aria-label]="VISUALIZATION"
-                              [data]="data"
-                              [(activeDatum)]="activeDatum"></app-line-chart-visualization>
-<app-line-chart-audification tabindex="0"
-                             [data]="data"
-                             [(activeDatum)]="activeDatum"></app-line-chart-audification>
+<app-card class="card" title="Line Chart">
```

Comment:
Should app-line-chart be a child of app-card (instead of vice versa)? It seems like the cards contain the visualizations.

Reply:
That makes sense. Would it be okay if I rename the component to `line-chart-card` instead?